ParisBNF/abb/BNFabb150.xml

```diff
@@ -99,7 +99,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             <item xml:id="q7" n="6">
                               <num value="6">፮</num>
                               <dim unit="leaf">7, stub after 7</dim>
```

Comment:
this is wrong, please check the guidelines as to where the specifications concerning stubs should go, thank you!

Reply:
> this is wrong, please check the guidelines as to where the specifications concerning stubs should go, thank you!

right! thank you!